Bottom: 1d8617026dd8983e62db33cfaa8dfb1abc313975
Top:    8f14cd095ce7ebb50e88ff3c91f0de39a9e4d282
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-12-23 23:37:34 +0000

Refresh of action-c-remove-change-log-fro

---

diff --git a/src/action.c b/src/action.c
index 69bf89d3..665ae91 100644
--- a/src/action.c
+++ b/src/action.c
@@ -1,5 +1,3 @@
-/* 15 Oct 2008 Ineiev: add CycleCrosshair action */
-
 /*
  *                            COPYRIGHT
  *
diff --git a/src/crosshair.c b/src/crosshair.c
index a644cc7..177c9cc 100644
--- a/src/crosshair.c
+++ b/src/crosshair.c
@@ -1,6 +1,4 @@
 /* $Id$ */
-/* 15 Oct 2008 Ineiev: add different crosshair shapes */
-
 /*
  *                            COPYRIGHT
  *
diff --git a/src/global.h b/src/global.h
index 8896901..029d63c 100644
--- a/src/global.h
+++ b/src/global.h
@@ -24,7 +24,6 @@
  *
  *  RCS: $Id$
  */
-/* 15 Oct 2008 Ineiev: add different crosshair shapes */
 
 /* definition of types
  */
diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index a4b34ee..c625fc2 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -1,5 +1,4 @@
 /* $Id$ */
-/* 15 Oct 2008 Ineiev: add different crosshair shapes */
 
 #ifdef HAVE_CONFIG_H
 #include "config.h"
